### ISSUE TITLE
Retrieve all SMS + Info

### ISSUE THREAD

**Issue by versaq** [AhMyth/AhMyth-Android-RAT#72]
Server should retrieve sent messages too, not just received ones.  Times and dates for messages and call logs would be nice too.

**Comment by max78612** [comment]
you are right. AndrorRAT is way better than it but is not stable. Ahmyth is much stable but very less features. In androRat you can listen to an audio from mic contineously
